models/Asset/Image/Thumbnail/Config.php

```diff
@@ -930,4 +930,34 @@ public function __clone()
             }
         }
     }
+
+    /**
+     * @return array
+     */
+    public static function getAutFormats(): array
+    {
+        return \Pimcore::getContainer()->getParameter('pimcore.config')['assets']['image']['thumbnails']['auto_formats'];
+    }
+
+    /**
+     * @return Config[]
+     */
+    public function getAutoFormatThumbnailConfigs(): array
+    {
+        $autFormatThumbnails = [];
```

Comment:
```suggestion
        $autoFormatThumbnails = [];
```
another typo, maybe double check your `o` key 😂

Reply:
Done...yes guess time to change my old keyboard.